index.js

```diff
@@ -37,31 +57,53 @@ function validate(word) {
 }
 
 function stonksify(word) {
-  //If a word starts with a c replace it with a k
-  //Example: cat => kat
+  let modifiedCharacters = Array.apply(null, Array(word.length)).map(
+    function() {
+      return 0;
+    }
+  );
+
```

Comment:
What does this do?

Reply:
this shouldn't have been in the commit and surprised me that it's here.

Reply:
will remove and re pull req